shotgun_api3/shotgun.py

```diff
@@ -66,19 +66,20 @@
 
 SG_TIMEZONE = SgTimezone()
 
-
 NO_SSL_VALIDATION = False
 try:
-    import ssl
-    if os.environ.get("SHOTGUN_DISABLE_SSL_VALIDATION", False):
+    import ssl        
+except ImportError, e:
+    if "SHOTGUN_FORCE_CERTIFICATE_VALIDATION" in os.environ:
+        raise ImportError("%s. SHOTGUN_FORCE_CERTIFICATE_VALIDATION environment variable prevents "
+                          "disabling SSL certificate validation." % e)
+    else:
```

Comment:
if `ssl` fails to import and you don't have `SHOTGUN_FORCE_CERTIFICATE_VALIDATION` in your environment variables, the else will kick in and we want to disable cert validation right?